plugins/wallabag.koplugin/main.lua

```diff
@@ -700,6 +716,46 @@ Restart KOReader after editing the config file.]]), DataStorage:getSettingsDir()
     self.settings_dialog:onShowKeyboard()
 end
 
+function Wallabag:editClientSettings()
+    self.client_settings_dialog = MultiInputDialog:new {
+        title = _("Wallabag client settings"),
+        fields = {
+            {
+                text = self.articles_per_sync,
+                description = T(_("Number of articles")),
```

Comment:
Btw, the description and the hint needn't be the same. 

Anyway, you're not using the template function here.
```suggestion
                description = _("Number of articles"),
```

Reply:
Right, thanks!

I also added a validation test to prevent crashes on incorrect values.